python/python-kafka-deployment.yaml

```diff
@@ -0,0 +1,59 @@
+apiVersion: kafka.strimzi.io/v1beta1
+kind: KafkaTopic
+metadata:
+  name: my-topic
+  labels:
+      strimzi.io/cluster: my-cluster
+spec:
+  replicas: 3
+  partitions: 12
+---
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  labels:
+    app: python-kafka-producer
+  name: python-kafka-producer
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      app: python-kafka-producer
+  template:
+    metadata:
+      labels:
+        app: python-kafka-producer
+    spec:
+      containers:
+        - name: python-kafka-producer
+          image: docker.io/owenerz/python-kafka-producer:latest
```

Comment:
Same as above

Reply:
Noted - will change to quay registry when I get nearer final submission.